src/reusecore/PageHeader/index.js

```diff
@@ -24,14 +45,31 @@ const PageHeader = ({ categories, title, subtitle,  author, thumbnail, path, sub
                         ))}
                         {author && (
                             <>
-                                <h4>By:</h4> <span>{author.name}</span>
+                                <h4>By:</h4> 
+                                { 
+                                    isSlugAvailable ? 
+                                        <Link className="authorLink"
+                                            to={`/community/members/${author.name.toLowerCase().replace(" ","-")}`}>
```

Comment:
just use the `slugify` function defined in `utils`

Reply:
Oh, thanks